### ISSUE TITLE
kops validate cluster doesn't show autoscale node count

### ISSUE THREAD

**Issue by sethpollack** [kubernetes/kops#2050]
My instance group is set to a min of 3 and a max of 10, when I run `kops validate cluster` I get this message:

```
Validation Failed
Ready Master(s) 3 out of 3.
Ready Node(s) 7 out of 3. 
```

**Comment by chrislovecnm** [comment]
Is this fixed now?

**Comment by sethpollack** [comment]
I don't think so

**Comment by chrislovecnm** [comment]
Please mark issue as you need, or it will close

**Comment by sethpollack** [comment]
Fixed in #4026
